collection_mutation.cc

```diff
@@ -153,6 +153,44 @@ api::timestamp_type collection_mutation_view::last_update(const abstract_type& t
     ));
 }
 
+std::ostream& operator<<(std::ostream& os, const collection_mutation_view::printer& cmvp) {
+    fmt::print(os, "{{collection_mutation_view ");
+    cmvp._cmv.with_deserialized(cmvp._type, [&os, &type = cmvp._type] (const collection_mutation_view_description& cmvd) {
+        bool first = true;
+        fmt::print(os, "tombstone {}", cmvd.tomb);
+        visit(type, make_visitor(
+        [&] (const collection_type_impl& ctype) {
+            auto&& key_type = ctype.name_comparator();
+            auto&& value_type = ctype.value_comparator();
+            for (auto&& [key, value] : cmvd.cells) {
+                if (!first) {
+                    fmt::print(os, ", ");
+                }
+                fmt::print(os, "{}: {}", key_type->to_string(key), atomic_cell_view::printer(*value_type, value));
+                first = false;
+            }
+        },
+        [&] (const user_type_impl& utype) {
+            for (auto&& e : cmvd.cells) {
```

Comment:
nit: `for (auto&& [raw_idx, val] : cmvd.cells) {`

Reply:
Yes.